partcad/src/partcad/runtime.py

```
@@ -28,3 +61,200 @@ def __init__(self, ctx, name):
+        cwd: str = None,
+        input_files: list[str] = None,
+        output_files: list[str] = None,
+    ):
+        if input_files is None:
+            input_files = []
+        if output_files is None:
+            output_files = []
+
+        if self.rpc_client:
+            file_contents = dict(
+                map(lambda x: (x, base64.b64encode(open(x, "rb").read()).decode("utf-8")), input_files)
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)